src/compiler/file_uploader/uploader_actor.ts

```diff
@@ -13,14 +13,13 @@ export async function createActor(
     return Actor.createActor<_SERVICE>(
         ({ IDL }) => {
             return IDL.Service({
-                clear_file_and_info: IDL.Func([IDL.Text], [], []),
-                get_file_hash: IDL.Func([IDL.Text], [IDL.Opt(IDL.Text)], []),
-                get_hash_status: IDL.Func(
+                _azle_clear_file_and_info: IDL.Func([IDL.Text], [], []),
+                _azle_get_file_hash: IDL.Func(
                     [IDL.Text],
-                    [IDL.Opt(IDL.Tuple(IDL.Nat64, IDL.Nat64))],
+                    [IDL.Opt(IDL.Text)],
```

Comment:
How was this working before?

Reply:
I think this is just a fluke of the git diff, when we added _azle to all of the methods it make _azle_get_file_hash too long to fit on one line. get_hash_status was removed entirely, and this git diff is mixing those two things up. Nothing changed here except adding _azle and removing get_hash_status. We no longer need get_hash_status since all the hashing is being done in one call